src/components/todoList.js

```diff
@@ -0,0 +1,128 @@
+import { actionTypes, progressTypes, filterTypes } from '../utils/constants.js';
+import createStore from '../store/store.js';
+import reducer from '../store/reducer.js';
+import todoCount from './todoCount.js';
+
+const todoList = (selectedFiter = filterTypes.ALL) => {
+  const store = createStore(reducer);
+  
+  const $listUl = document.getElementById('todo-list');
+
+  function saveTodoToLocal() {
+    const state = store.getState();
+    localStorage.setItem('todoState', JSON.stringify(state));
+  }
+  
+  function render() {
+    const state = store.getState();
+    let showItems = { ...state };
+
+    Object.values(state).map(item => {
+      if(selectedFiter === filterTypes.ACTIVE && item.completedFlag) {
+        delete showItems[item.seq];
+        return showItems;
+      } else if(selectedFiter === filterTypes.COMPLETED && !item.completedFlag) {
+        delete showItems[item.seq];
+        return showItems;
+      }
+    });
+
+    const itemList = Object.values(showItems).map(obj => {
+      return (`
+      <li id="item-${obj.seq}" class="${obj.completedFlag === false ? (obj.editFlag === false ? '': progressTypes.EDITING) : progressTypes.COMPLETED}">
+        <div class="view">
+          <input class="toggle" type="checkbox" ${obj.completedFlag === true? 'checked' : ''}/>
+          <label class="label">${obj.content}</label>
+          <button class="destroy"></button>
+        </div>
+        <input class="edit" value=${obj.content} />
+      </li>
+      `)
+      }).join("");
+    $listUl.innerHTML = itemList;
+
+    bindEvent(showItems);
+    onEdit();
+    todoCount(showItems);
+    saveTodoToLocal();
+  };
+
+  function bindEvent(showItems) {
+    Object.values(showItems).map(obj => {
+      const $liItem = document.getElementById("item-"+obj.seq);
+      
+      $liItem.addEventListener('click', e => {
```

Comment:
해당 이벤트에서 ```event```의 요소 중 ```target``` 속성만 사용하는데 다음과 같이 구조분해할당을 통해 ```target```만 받아왔으면 어땠을까 하는 생각이 드네요!
```suggestion
      $liItem.addEventListener('click', { target } => {
```